.github/workflows/golangci-lint.yml

```diff
@@ -0,0 +1,41 @@
+name: golangci-lint
+on:
+  pull_request:
+permissions:
+  contents: read
+  # Optional: allow read access to pull request. Use with `only-new-issues` option.
+  # pull-requests: read
+
+jobs:
+  golangci:
+    name: lint
+    runs-on: ubuntu-latest
```

Comment:
So this will not catch conditional windows/mac only files for lining? Not sure how much we have anymore.

Reply:
I don't think there's much. I think most of what we have is for linux only. I'm ok with missing a few things.